<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWinClass</class>
 <widget class="QMainWindow" name="mainWinClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoForMe!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwin.qrc">
    <normaloff>:/mainWin/logo.png</normaloff>:/mainWin/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="scriptsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>23</y>
      <width>151</width>
      <height>198</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget::item::selected {
	background-color: rgb( 51, 153, 255 );
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb( 210, 210, 210 );
color: black;</string>
    </property>
    <property name="text">
     <string>  Scripts</string>
    </property>
   </widget>
   <widget class="QToolButton" name="removeScriptButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>1</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb( 210, 210, 210 );
color: black;</string>
    </property>
    <property name="text">
     <string>  Actions</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="scriptTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>23</y>
      <width>661</width>
      <height>198</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid rgb( 130, 130, 130 );
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="scriptTitle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>661</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb( 210, 210, 210 );
color: black;</string>
    </property>
    <property name="text">
     <string>  New script</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="actionsTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>252</y>
      <width>151</width>
      <height>209</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
selection-background-color: rgb( 51, 153, 255 );
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Script</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>661</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb( 210, 210, 210 );
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="editButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>231</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="detachButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>401</x>
      <y>231</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Detach</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="removeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>231</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="saveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>231</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveOnCloseCheck">
    <property name="geometry">
     <rect>
      <x>723</x>
      <y>233</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Save on close</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="addActionButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>231</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add to actions</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="leftButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>231</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwin.qrc">
      <normaloff>:/arrows/Resources/1leftarrow.ico</normaloff>:/arrows/Resources/1leftarrow.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="rightButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>231</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwin.qrc">
      <normaloff>:/arrows/Resources/1rightarrow.ico</normaloff>:/arrows/Resources/1rightarrow.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="upButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>231</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwin.qrc">
      <normaloff>:/arrows/Resources/1uparrow.ico</normaloff>:/arrows/Resources/1uparrow.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="downButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>231</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="mainwin.qrc">
      <normaloff>:/arrows/Resources/1downarrow.ico</normaloff>:/arrows/Resources/1downarrow.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCustomize">
    <property name="title">
     <string>Customize</string>
    </property>
    <addaction name="actionCalendar_colours"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionMenuNewAction"/>
    <addaction name="separator"/>
    <addaction name="actionMenuEditAction"/>
    <addaction name="actionMenuRemoveAction"/>
    <addaction name="actionMenuDetachAction"/>
    <addaction name="separator"/>
    <addaction name="actionMenuMoveLeft"/>
    <addaction name="actionMenuMoveUp"/>
    <addaction name="actionMenuMoveDown"/>
    <addaction name="actionMenuMoveRight"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionMenuInterval"/>
    <addaction name="actionMenuRecorder"/>
    <addaction name="actionMenuReminder"/>
   </widget>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>Script</string>
    </property>
    <addaction name="actionMenuNewScript"/>
    <addaction name="actionMenuImportScripts"/>
    <addaction name="separator"/>
    <addaction name="actionMenuSaveScript"/>
    <addaction name="actionMenuRemoveScript"/>
    <addaction name="separator"/>
    <addaction name="actionMenuParseScript"/>
    <addaction name="actionMenuRunScript"/>
   </widget>
   <addaction name="menuScript"/>
   <addaction name="menuActions"/>
   <addaction name="menuCustomize"/>
   <addaction name="menuView"/>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>26</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionImport"/>
   <addaction name="actionSaveScript"/>
   <addaction name="actionTray"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionParseScript"/>
   <addaction name="actionRun"/>
   <addaction name="actionRecord"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExlude_commands">
   <property name="text">
    <string>Exclude...</string>
   </property>
  </action>
  <action name="actionCalendar_colours">
   <property name="text">
    <string>Calendar colors...</string>
   </property>
  </action>
  <action name="actionMenuNewAction">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionScript_code">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Script code</string>
   </property>
  </action>
  <action name="actionRecording">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recording</string>
   </property>
  </action>
  <action name="actionTiming">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timing</string>
   </property>
  </action>
  <action name="actionRepeating">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeating</string>
   </property>
  </action>
  <action name="actionReminding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reminding</string>
   </property>
  </action>
  <action name="actionMenuEditAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionMenuRemoveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionMenuDetachAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detach</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/play_record/Resources/play_on.png</normaloff>:/play_record/Resources/play_on.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/play_record/Resources/record_on.png</normaloff>:/play_record/Resources/record_on.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="toolTip">
    <string>Record</string>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/standard/Resources/New.ico</normaloff>:/standard/Resources/New.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New script</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/standard/Resources/folder_new.ico</normaloff>:/standard/Resources/folder_new.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open script</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSaveScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/standard/Resources/filesave.ico</normaloff>:/standard/Resources/filesave.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionTray">
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/mainWin/Resources/tray_icon.png</normaloff>:/mainWin/Resources/tray_icon.png</iconset>
   </property>
   <property name="text">
    <string>Hide to tray</string>
   </property>
   <property name="toolTip">
    <string>To tray</string>
   </property>
  </action>
  <action name="actionMenuMoveLeft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move left</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionMenuMoveUp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="actionMenuMoveDown">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionMenuMoveRight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move right</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionMenuNewScript">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionMenuRemoveScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionMenuRunScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMenuSaveScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMenuImportScripts">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionParseScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwin.qrc">
     <normaloff>:/play_record/Resources/parse_script.png</normaloff>:/play_record/Resources/parse_script.png</iconset>
   </property>
   <property name="text">
    <string>Parse script</string>
   </property>
   <property name="toolTip">
    <string>Parse script</string>
   </property>
  </action>
  <action name="actionMenuParseScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Parse</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionMenuRecorder">
   <property name="text">
    <string>Recorder...</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionMenuReminder">
   <property name="text">
    <string>Reminder...</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionMenuInterval">
   <property name="text">
    <string>Player...</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>saveOnCloseCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
